src/api/phone.js

```diff
@@ -1,18 +1,34 @@
-export const getById = (phoneId) => {
-  return {
-    "description": "Feel Free to Tab\u2122. The Samsung Galaxy Tab\u2122, the tablet device that delivers enhanced capabilities with advanced mobility, has a large, perfectly sized, 7.0\" screen that offers plenty of room for the thousands of interactive games and apps available for the Android\u2122 platform, and its slim design makes it perfect for travel and one-handed grip. Use the Galaxy Tab to relax and enjoy an e-book, watch rich video or full web content with its Adobe\u00ae Flash\u00ae Player compatibility, video chat using the front-facing camera, or send user-generated content wirelessly to other devices like your TV via AllShare\u2122.  With so many options for customization and interactivity, the Galaxy Tab gives you everything you want, anywhere you go\u2026Feel Free to Tab\u2122.",
-    "id": "samsung-galaxy-tab",
-    "images": [
-      "img/phones/samsung-galaxy-tab.0.jpg",
-      "img/phones/samsung-galaxy-tab.1.jpg",
-      "img/phones/samsung-galaxy-tab.2.jpg",
-      "img/phones/samsung-galaxy-tab.3.jpg",
-      "img/phones/samsung-galaxy-tab.4.jpg",
-      "img/phones/samsung-galaxy-tab.5.jpg",
-      "img/phones/samsung-galaxy-tab.6.jpg"
-    ],
-    "name": "Samsung Galaxy Tab\u2122",
-  };
+export const getById = async (data, phoneId) => {
+ 
+  const [ selectedPhone ] = data.filter(phone => phone.id === phoneId),
+        id = `${selectedPhone.id}`,
+        imagePaths = [];
+
+  for (let i = 0; i < 6; i++){
+    const src = `img/phones/${id}.${i}.jpg`,
+          checkImage = (path) =>
```

Comment:
```suggestion
  const  checkImage = (path) =>
```
Split const declaration to multiple lines. It's easier to read.

Reply:
done